test/testwebkitserver.pro

```diff
@@ -1,5 +1,5 @@
 SOURCES = testignoredebugoutput.cpp
-OBJECTS += ../src/IgnoreDebugOutput.o
+OBJECTS += ../src/build/IgnoreDebugOutput.o
```

Comment:
This wasn't necessary in 5.5, but it fails for me with 5.6. I'm not sure if it would work in earlier versions of Qt.

If we reference `src/IgnoreDebugOutput.o`, it actually tries to build that from `src/IgnoreDebugOutput.cpp`, which fails because the `QtGlobal` header isn't available when it's building test dependencies.

Referencing `src/build/IgnoreDebugOutput.o` links against the already-built object.